code/game/objects/items/ego_weapons/he.dm

```diff
@@ -1784,3 +1784,50 @@
 		poise = 0
 	..()
 	force = initial(force)
+
+/obj/item/ego_weapon/uturn
+	name = "u-turn"
+	desc = "It's a large scythe, that probably hurts a lot."
+	icon_state = "uturn"
+	force = 30
+	damtype = RED_DAMAGE
+	attack_verb_continuous = list("whips", "lashes", "tears")
+	attack_verb_simple = list("whip", "lash", "tear")
+	hitsound = 'sound/weapons/whip.ogg'
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 40
+							)
+	var/can_spin = TRUE
+	var/spinning = FALSE
+
+/obj/item/ego_weapon/uturn/attack(mob/living/target, mob/living/user)
+	if(spinning)
+		return FALSE
+	..()
+	can_spin = FALSE
+	addtimer(CALLBACK(src, .proc/spin_reset), 12)
+
+/obj/item/ego_weapon/uturn/proc/spin_reset()
+	can_spin = TRUE
+
+/obj/item/ego_weapon/uturn/attack_self(mob/user)
+	if(!CanUseEgo(user))
+		return
+	if(!can_spin)
+		to_chat(user,"<span class='warning'>You attacked too recently.</span>")
+		return
+	if(do_after(user, 12, src))
+		can_spin = TRUE
+		addtimer(CALLBACK(src, .proc/spin_reset), 12)
+		playsound(src, 'sound/weapons/ego/harvest.ogg', 75, FALSE, 4)
+		for(var/turf/T in orange(1, user))
+			new /obj/effect/temp_visual/smash_effect(T)
+
+		for(var/mob/living/L in range(1, user))
+			var/aoe = 30
```

Comment:
```suggestion
			var/aoe = force
```
Lets us do cool things

Reply:
I that's the case - there's no need to use aoe var at all, just put force in damage place.